### PR TITLE
skin grafts and surgical sutures (from CM)

### PR DESCRIPTION
**Pull request by pedr69**
### Have you ever found a dead body with too much damage to defibrillate?
image

fear no more! our crate pushers have obtained military grade medical gear from another space sector!
available from cargo for 50 credits, you may now stitch pulped/burned spacemen together again using simple surgery!
after locating a surgical table(or a normal one), even YOU can become a savior of lives!
image
<sub><sub>this message brought to you by the Cargonian Union. we are not responsible for any injuries caused by suturing performed by non medical personnel. sutures do not mend bones. sutures do not stop bleeding, either external or internal. consult your practitioner before use.</sub></sub>

## What this does
adds synthgrafts and surgical line
they can be used to heal brute/burn damage on heavily injured limbs

## Why it's good
ummmm....

## Changelog

 * rscadd: skin grafts and surgical line, for treating heavily damaged limbs in surgery. available at cargo!
 * imageadd: synthgrafts and surgical line

### CODE REVIEW

**Comment by SonixApache** [comment]
+1 for not having to do medical/surgery shitcode to achieve the same result anymore

**Comment by Eneocho** [comment]
I'd say this fits better as part of the advanced trauma/burn kits instead of their own tool.

**Comment by Hinaichigo** [comment]
The trauma and burn kits seem a little too black-boxey to me. It'd be nice to have to actually stitch people up, and maybe there could be ghetto suturing and stuff.

**Comment by pedr69** [comment]
>advanced kits

this is more of a complement to the kits, it doesn't stop bleeding or treat the wound, so after you stitch em up you need to slap a traumakit on top to heal(or use pills)
>too blackboxey

not sure what you mean, a scalpel step before applying the thing?

**Comment by Hinaichigo** [comment]
With the kits, its moreso that it's not exactly clear whats in the kit, meanwhile the rest of the surgery has all of the different tools and everything explicitly for each step. So I like this change because it's more nitty-gritty.

**Comment by jwhitak** [comment]
I'm not really sure I like this addition, since it removes the soul of medbay removing limbs and committing strange rituals to heal the husked dead...

But meh it makes things easier. In fact, my only real complaint is that a few of these relatively low tech medicinal items don't spawn in medical storage at round start. They seem common enough that they'd be available. Not a deal breaker by any means. I'm happy to start with a cargo buff and see where things go from there.

**Comment by pedr69** [comment]
i was thinking either cargo or the fabled chemistry vial synthesis reaction
settled for cargo, more player interaction is better than reading the wiki in the chemistry room

**Comment by Hinaichigo** [comment]
It makes things easier in some sense but it also paves the way for the holy grail; the cloning nerf.

**Comment by Eneocho** [comment]
Keep it as an expensive cargo request for fast and large quantities but you blow your budget, but add a vial option that uses clonex, bicaridine and sacid to make a single use synthetic skin graft.

**Comment by west3436** [comment]
>pay-to-win corpse healing tools

**Comment by Eneocho** [comment]
> >pay-to-win corpse healing tools

we already have the pay-to-win bombs (shards), pay-to-win armor, pay-to-win guns, etc

**Comment by Eneocho** [comment]
> > pay 2 heal
> 
> not a bad idea actually autodoc machine that requires money to treat your IB emagged ones remove your kidneys and give you some money for them instead

That's sorta a thing already if you find a pay2use beno artifact that heals. I've seen em once or twice.

File: code/game/objects/items/weapons/surgery_sutures.dm [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/obj/item/tool/suture/ //parent type, should never appear ingame
+	name = "Suture"
+	desc = "Some sort of simple surgical tool. this one doesn't seem to do anything."
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/surgery_sutures.dm]: This (I realize it's an abstract type but come on)

File: code/game/objects/items/weapons/surgery_sutures.dm [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	heal_brute = 10
+	heal_burn = 0
+
+
+//copied from CM-ss13
+//a simple surgical tool, used to repair heavy burns
+//TODO make it actually change your skin color
+/obj/item/tool/suture/synthgraft
+	name = "\improper Synth-Graft"
+	desc = "An applicator for synthetic skin field grafts. The stuff reeks, itches like the dickens, hurts going on, and the colour is \
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/surgery_sutures.dm]: weird line break, just remove it

File: code/game/objects/items/weapons/surgery_sutures.dm (hunk 1 above)
Comment (pedr69) [reply code/game/objects/items/weapons/surgery_sutures.dm:3]: this what?
what do you want me to do?
the graft and suture not be subclasses of a suture?

Comment (Kurfursten) [reply code/game/objects/items/weapons/surgery_sutures.dm]: You didn't capitalize the word "This"

**Pull request merged**